cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -20,15 +20,21 @@ func getStateCommitment(commits storage.Commits, blockHash flow.Identifier) (flo
 
 func extractExecutionState(dir string, targetHash flow.StateCommitment, outputDir string, log zerolog.Logger) error {
 
-	led, err := complete.NewLedger(dir, complete.DefaultCacheSize, &metrics.NoopCollector{}, log, nil, 0)
+	led, err := complete.NewLedger(
+		dir,
+		complete.DefaultCacheSize,
+		&metrics.NoopCollector{},
+		log,
+		nil,
+		complete.DefaultPathFinderVersion)
 	if err != nil {
 		return fmt.Errorf("cannot create ledger from write-a-head logs and checkpoints: %w", err)
 	}
 	filePath := path.Join(outputDir, "root.checkpoint")
 
 	newState, err := led.ExportCheckpointAt(targetHash,
-		[]ledger.Migration{migrations.MultipleContractMigration, migrations.AddMissingKeysMigration},
```

Comment:
No. These were already applied on the testnet, and currently the only way to not run them again is to remove them here. 

If the future it would be great to have detection if a migration has already been applied, so removing it is not needed.

The branch that is going to be deployed on mainnet is already separated from the master branch, so this won't affect mainnet deployment if merge into master.

Reply:
it can be removed from here, we keep the migration code but we won't need them as part of execution state extract anymore